### DISCUSSION TITLE
Set Tycho version for tycho-versions-plugin

### DISCUSSION THREAD

**Discussion by globavi** [eclipse-tycho/tycho#1480] [Q&A] [answered]
Hi,

I tried to set the version for org.eclipse.tycho:tycho-versions-plugin:set-version in the project's plugin configuration as described here. However it always takes the newest one (3.0.0) instead the one I configured there.
https://ci.eclipse.org/tycho/job/tycho-sitedocs/lastSuccessfulBuild/artifact/target/staging/tycho-release/tycho-versions-plugin/plugin-info.html

I use as command ~
`mvn -f pom.xml org.eclipse.tycho:tycho-versions-plugin:set-version -Pall -DnewVersion=*** -Dartifacts=root`

and configured in the project settings 
       
```
 <plugin>
          <groupId>org.eclipse.tycho</groupId>
          <artifactId>tycho-versions-plugin</artifactId>
          <version>{version}</version> ##set to 2.7.5
   </plugin>
```

The "workaround" I found is to use org.eclipse.tycho:tycho-versions-plugin:2.7.5:set-version what is however duplicate code. I use afterwards another maven command in which the version from the settings (extensions) is used.

**Comment by laeubi** [comment, marked as answer]
As mentioned before, when you call the mojo directly from the commandline your pom configuration is not considered!

**Reply by globavi** [reply]
i see. thx again (I guess i have then to use the "workaround" as the goal das not bind to a phase)
